### PR TITLE
scrabble-score: add new exercise

### PR DESCRIPTION
**Pull request by filiptoskovic**

### CODE REVIEW

File: exercises/scrabble-score/uScrabbleScoreExample.pas [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+type
+  TScrabble = class
+  private
+    const
+      FValues : array ['A' .. 'Z'] of integer =
+        (1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10);
+  public
+    class function Score(AInp : string) : integer;
+  end;
```
Comment (rpottsoh) [inline exercises/scrabble-score/uScrabbleScoreExample.pas]: You don't have to make this change if you don't want to, this is just an example solution.  I just wanted to show you an alternative way to write this that is a little cleaner.
```pascal
type
  TScrabble = class
  private
    type
      TAlphaVals = array['A'..'Z'] of integer;
    const
      FValues : TAlphaVals = (1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10);
  public
    class function Score(AInp : string) : integer;
  end;
```

**Comment by rpottsoh** [comment]
Nice work.  I'll work on the configuration.

**Comment by rpottsoh** [comment]
You're merging again....  remember you fast forward from Origin to Local, or from remote to Local.  You push from local to origin.  Don't need to merge.

I am going to be rebasing scrabble-score on to master so the config.json changes made in series will be realized, or that is my plan anyway.

**Comment by rpottsoh** [comment]
You need to fast forward from origin to local to update your local copy of the branch.  You also need to fast forward from remote master to local master and then push to origin master to update your master

**Comment by rpottsoh** [comment]
Let me know when you think you are done and then I will merge.

**Comment by filiptoskovic** [comment]
I think I'm done

**Pull request merged**